pkgs/development/python-modules/pylibgen/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, lib, fetchFromGitHub
+, isPy3k, isPyPy
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pylibgen";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "JoshuaRLi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0mzf5741xp8lgh6q3m041a64wlwwyagf9226rw5aixnd5h1r4cic";
+  };
+
+  # Python 2.x is not supported.
+  disabled = !isPy3k && !isPyPy;
```

Comment:
So, Python 2 isn't supported but PyPy is?

Reply:
my fault